### PR TITLE
SAK-32383 Tidied up the profile popup.

### PR DESCRIPTION
**Pull request by adrianfish**
Make the default texts and tool tips clearer. Add some spacing under the
icons and bold the icon text.

### CODE REVIEW

File: portal/portal-impl/impl/src/bundle/profile-popup.properties [hunk 1]
```diff
@@ -2,18 +2,18 @@ connect = Connect
...
 
 profile = Profile
 profile_tooltip = Click on this to view this person's Profile
 email = Email
 email_tooltip = Click on this to send this person an email
 facebook_tooltip = Click on this to launch this person's Facebook page
-no_facebook_tooltip = This person has not setup a Facebook link
-twitter_tooltip = Click on thie to launch this person's Twitter feed
-no_twitter_tooltip = This person has not setup a Twitter link
+no_facebook_tooltip = There is no Facebook account on this profile
```
Comment (ottenhoff) [inline portal/portal-impl/impl/src/bundle/profile-popup.properties]: "on this profile" doesn't seem naturally phrased to me

Comment (adrianfish) [inline portal/portal-impl/impl/src/bundle/profile-popup.properties]: How about 'attached to this profile'? The aim's to just clue someone in where to go to add it.

Comment (ottenhoff) [inline portal/portal-impl/impl/src/bundle/profile-popup.properties]: yes, attached to makes more sense

Comment (adrianfish) [inline portal/portal-impl/impl/src/bundle/profile-popup.properties]: Done.

**Comment by adrianfish** [comment]
More code coming ...

**Comment by adrianfish** [comment]
@ottenhoff Can I merge this in?

**Pull request merged**